MeshLib/IO/GmshReader.h

```diff
@@ -0,0 +1,53 @@
+/**
+ *
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include <fstream>
+#include <map>
+#include <string>
+#include <vector>
+
+namespace MeshLib
+{
+class Node;
+class Element;
+class Mesh;
+
+namespace IO
+{
+namespace GMSH
+{
+
+void readNodeIDs(std::ifstream& in, unsigned n_nodes,
```

Comment:
Is it right that  this function should not be called from outside? If that's the case, do you still need the declaration here? same applies to the below function